### ISSUE TITLE
Errors running NPM commands after posting React app to github pages.

### ISSUE THREAD

**Issue by TS22082** [gitname/react-gh-pages#16]
Everything was working before deploying to github pages, and the first deploy worked no problem. But ever since, now when running any NPM commands in the project, I'm getting this set of errors:

```
geekcentric => npm start
npm ERR! file /mnt/c/Users/GeekCentric/Documents/ReactJS/the_app/package.json
npm ERR! code EJSONPARSE
npm ERR! JSON.parse Failed to parse json
npm ERR! JSON.parse Unexpected token / in JSON at position 611 while >parsing near '...": "^1.2.0"
npm ERR! JSON.parse }
npm ERR! JSON.parse }
npm ERR! JSON.parse
npm ERR! JSON.parse
npm ERR! JSON.parse // geekcentric => np...'
npm ERR! JSON.parse Failed to parse package.json data.
npm ERR! JSON.parse package.json must be actual JSON, not just >JavaScript.
npm ERR! A complete log of this run can be found in:
npm ERR! /home/geekcentric/.npm/_logs/2018-09-11T00_38_07_636Z->debug.log
```

The error is in my package.json file but I can't find anything that would throw the error or what's going wrong.

```
{
  "name": "theapp",
  "version": "0.1.0",
  "private": true,
  "homepage": "http://me.github.io/theapp",
  "dependencies": {
    "axios": "^0.18.0",
    "firebase": "^5.4.2",
    "react": "^16.4.2",
    "react-dom": "^16.4.2",
    "react-router-dom": "^4.3.1",
    "react-scripts": "1.1.5"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test --env=jsdom",
    "eject": "react-scripts eject",
    "predeploy": "npm run build",
    "deploy": "gh-pages -d build"
  },
  "devDependencies": {
    "gh-pages": "^1.2.0"
  }
}
```

I'm using the create-react-app and the gh-pages npm module. Any help would be very appreciated

**Comment by TS22082** [comment]
I updated my text editor (VSCODE) and its working fine now. I use the built-in terminal with WSL so I can only imagine I inadvertently screwed up a critical path. Still really curious to know what happened and why only after hosting on gh-pages but the errors are gone and everything is running as you would expect.

**Comment by shihisthebest** [comment]
@TS22082 I know you encountered this issue a couple years back, but just wanted to flag that I recently ran into the same issue! I developed my React app and tested it at localhost:3000 up until I successfully pushed it to Github Pages, at which point I could no longer execute any npm commands. See attached for example.

If anyone has successfully figured this issue out, I'd love to know your workaround! It's like hearing the setup of a great joke and then having the teller walk away before they deliver the punch line...

![error1](https://user-images.githubusercontent.com/62533535/104996427-b9cf9b80-59dc-11eb-9e1f-eed88fdac14b.PNG)
![error2](https://user-images.githubusercontent.com/62533535/104996430-ba683200-59dc-11eb-953b-bcf8402f2646.PNG)

!

**Comment by shihisthebest** [comment]
@TS22082 I know you encountered this issue a couple years back, but just wanted to flag that I recently ran into the same issue! I developed my React app and tested it at localhost:3000 up until I successfully pushed it to Github Pages, at which point I could no longer execute any npm commands. See attached for example.

If anyone has successfully figured this issue out, I'd love to know your workaround! It's like hearing the setup of a great joke and then having the teller walk away before they deliver the punch line...
![error2](https://user-images.githubusercontent.com/62533535/104996701-32cef300-59dd-11eb-87c1-337a81061e0d.PNG)
![error1](https://user-images.githubusercontent.com/62533535/104996704-32cef300-59dd-11eb-9eb4-9ceadb513c3f.PNG)
